spec/dummy/config/application.rb

```diff
@@ -12,8 +12,8 @@
 Bundler.require(:puma_min) if !!ENV['PUMA_MIN']
 Bundler.require(:thin_max) if !!ENV['THIN_MAX']
 Bundler.require(:thin_min) if !!ENV['THIN_MIN']
-Bundler.require(:unicorn_min) if !!ENV['UNICORN_MIN_TEST']
-Bundler.require(:unicorn_max) if !!ENV['UNICORN_MAX_TEST']
```

Comment:
Might as well also remove `UNICORN_MAX_TEST` from `docker/Dockerfile_unicorn_max`, and `UNICORN_MIN_TEST` from `docker/Dockerfile_unicorn_min` just to keep things tidy :)

Reply:
Good call 👍